dist/index.js

```diff
@@ -616,10 +613,14 @@ function createPipeline(config) {
             yield validatePipeline(pipeline);
             core.debug(`Sending pipeline: ${util_1.default.inspect(pipeline)}`);
             //TODO: Define and replace different placeholders: e.g. for values, content folders (goss, jmeter), etc.
+            const expiresAfter = (0, moment_1.default)()
+                .add(config.pipelineDuration * 1000, "s")
+                .format("ddd, D MMM YYYY HH:mm:ss z");
```

Comment:
```suggestion
                .format("ddd, DD MMM YYYY HH:mm:ss z");
```

`DD` for days to be 01, 02, ..

Reply:
done